### ISSUE TITLE
Very small bug in gui

### ISSUE THREAD

**Issue by drlauridsen** [gskjold/AmsToMqttBridge#323]
Minor bug in GUI.

When choosing any page, the ENTSO-E API link disappears from the menu.
The pagelink is shown only from the main page, and can be shown again simply by going back to the main page.

![Screenshot_20221008_170332](https://user-images.githubusercontent.com/61352332/194714208-788aa067-99bc-4336-a793-4dba9563ed58.jpg)
![Screenshot_20221008_170315](https://user-images.githubusercontent.com/61352332/194714210-47292894-f1fa-43c8-803a-46d0fa83c5c8.jpg)

**Comment by ArnieO** [comment]
Yeah - I've seen this too, but have forgotten to report it. Thank you!

**Comment by gskjold** [comment]
Sorry I have also seen this, but have ignored it.. Will improve :)

**Comment by gskjold** [comment]
Fixed in upcoming version 2.2.0

**Comment by gskjold** [comment]
v2.2 beta out for testing now:
https://github.com/gskjold/AmsToMqttBridge/discussions/370
